core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/Generated.java

```diff
@@ -0,0 +1,15 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+/**
+ * Marker for all classes generated by annotation processor.
```

Comment:
I'm yet to review other parts, but my understanding is that this annotation is only for intermediate usage during code generation. Should we explicitly mention that this annotation isn't expected to be used in the codebase?

also, nit: this file is the only file that uses static import for TYPE and SOURCE, probably you want to change it to usual import to be consistent with other classes.

Reply:
Comment added!

I just rely on the automatic cleanups for imports in IDEA. It would be a C++ nightmare to maintain all imports manually :)